manager/src/main/java/com/redhat/service/smartevents/manager/dao/ProcessorDAO.java

```diff
@@ -182,4 +182,11 @@ public Long countByBridgeIdAndCustomerId(String bridgeId, String customerId) {
         return countProcessorsOnBridge(p);
     }
 
+    public Long countByBridgeId(String bridgeId) {
+        TypedQuery<Long> namedQuery = getEntityManager().createNamedQuery("PROCESSOR.countByBridgeId", Long.class);
```

Comment:
This will return the count of [ALL](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/885/files/344eca33bc484198a791e76cb67baaac9a4865d6#diff-5d407c38031fef17a0861a9d9ea1141ba31a15b29b463de5b160b03b3b204057R63-R64) `Processor`'s including Error Handlers.

Reply:
Good catch @manstis

Reply:
I added the filters to count processors of type SINK and SOURCE only.